frontend/src/components/dropdown.vue

```diff
@@ -0,0 +1,197 @@
+<template>
+<li @click="visible = !visible;" class="dropdown-main-item"><a class="dropdown-icon"><img src="~../assets/menu.svg" class="dropdown-icon"></a>
+<ul v-if="visible" class="dropdown-menu" style="display: inline-block;">
+    <li id="dropdown-submenu-item_styling" class="dropdown-submenu-item">
+        <p class="dropdown-menu-item_p">Information</p>
+        <router-link style="text-decoration: none; color: #333333;" to="/about">
+        <div id="darkmode-icon" class="dropdown-submenu-item_div">
+            <p class="dropdown-submenu-subitem">About</p>
+        </div>
+        </router-link>
+    </li>
+    <li id="dropdown-menu-item_shuttle-schedule" class="dropdown-menu-item">
+        <p class="dropdown-menu-item_p">Shuttle Schedules</p>
+        <ul id="dropdown-submenu_shuttle-schedule" class="dropdown-submenu" style="display: inline-block;">
+            <li id="dropdown-submenu-item_shuttle-schedule" class="dropdown-submenu-item">
+                <p class="dropdown-submenu-item_p"><a target="_blank" rel="noopener noreferrer" href="http://www.rpi.edu/dept/parking/shuttle/2017-2018CampusShuttleScheduleEastRoute.pdf"
```

Comment:
These links are broken. However, I assume the changes that Grace made when she fixed that will be merged... ?